spec/core/GraphErrorHandler.ts

```diff
@@ -97,9 +97,9 @@ describe("GraphErrorHandler.ts", () => {
 
 		it("Should construct some default error", async () => {
 			const gError = await GraphErrorHandler.getError();
+			assert.equal(gError.message, "");
 			assert.equal(gError.statusCode, -1);
 			assert.equal(gError.code, null);
-			assert.equal(gError.message, null);
 			assert.equal(gError.body, null);
 			assert.equal(gError.requestId, null);
 		});
```

Comment:
Can you please add unit tests verifying if stack is set or defined as expected as per the new line added in this PR -> 
`this.stack = baseError ? baseError.stack : this.stack;` ?

Reply:
probably need an issue to track adding a stack assertion for each of the cases here.